### PR TITLE
[Bug] Fixed StyleCop Errors

### PR DESCRIPTION
**Pull request by BraedonWooding**
### The issue this fixes
Some stylecop issues must have leaked through the last few commits so here is the fixes to those issues.

### What this PR does
Fixes all the spaces after some of the headers (the license headers).  It also renames zLevel to currentLayer, since it's called currentLayer everywhere it is used, and zLevel is not a good naming convention (raised a stylecop error even) and currentLayer is more descriptive.

### CODE REVIEW

File: Assets/Scripts/Models/InputOutput/CameraData.cs [hunk 1]
```diff
@@ -15,14 +14,14 @@ public struct Preset
...
-    public int zLevel;
+    public int currentLayer;
 }
 
 public class CameraData
 {
     public Vector3 position;
     public float zoomLevel;
-    public int zLevel;
+    public int currentLayer;
```
Comment (bjubes) [inline Assets/Scripts/Models/InputOutput/CameraData.cs]: Does anyone have better potential names for this variable? I think "Layer" is a bit to similar to "overlay", but currentLevel isn't great either?
maybe depthLayer or depthLevel?

**Comment by koosemose** [comment]
currentLayer is incorrect, it isn't the current Layer.

**Comment by BraedonWooding** [comment]
Then what should it be called?  zLevel is a bad name?

**Comment by bjubes** [comment]
#1617  fixes the hungarian notation error so revert the variable name to `zLevel`. All the other changes are valid stylecop violations.

**Comment by BraedonWooding** [comment]
Made the changes requested 😄

**Pull request merged**